.travis.yml

```diff
@@ -0,0 +1,43 @@
+language: python
+sudo: false
+
+services:
+  - docker
+
+env:
+  global:
+    - GH_REF: github.com/NSLS-II/scipy-binder.git
+
+cache:
+  directories:
+    - $HOME/.cache/pip
+    - $HOME/.ccache
+
+matrix:
+  fast_finish: true
+  include:
+    - python: 3.7
+      env: BUILD_DOCS=true SUBMIT_CODECOV=true NUMPY=1.17
+
+python:
+  - 3.7
+
+env:
+  - NUMPY=1.17
+
+before_install:
```

Comment:
I think I mentioned as I was thinking this through that we could install conda this way. But, as I continued thinking it through, I realized that we really want to test that `repo2docker  .` works end to end, and that will take care of running conda for us (inside a docker container). Thus I think this is unnecessary.